daemon/mgr/container_types.go

```
@@ -154,6 +156,27 @@ func (meta *ContainerMeta) merge(getconfig func() (v1.ImageConfig, error)) error
 	return nil
 }
 
+// FormatStatus format container status
+func (meta *ContainerMeta) FormatStatus() (string, error) {
+	var status string
+
+	if meta.State.Status == types.StatusRunning || meta.State.Status == types.StatusPaused {
+		start, _ := time.Parse(utils.TimeLayout, meta.State.StartedAt)
```

Please do not ignore this error.